Resources/Prototypes/_Lust/Entities/Clothing/Neck/collar.yml

```
@@ -1,22 +1,16 @@
 - type: entity
-  parent: Clothing
+  parent: ClothingNeckBase
   id: ClothingNeckShockCollar
   name: shock collar
   suffix: SelfUnremovable
   description: An electric collar that shocks on the signal.
   components:
-  - type: Item
-    size: Small
   - type: Sprite
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)